packages/yew/src/virtual_dom/vcomp.rs

```diff
@@ -14,27 +14,25 @@ use super::Key;
 use crate::dom_bundle::BSubtree;
 #[cfg(feature = "hydration")]
 use crate::dom_bundle::Fragment;
-#[cfg(feature = "csr")]
-use crate::html::Scoped;
+use crate::html::BaseComponent;
 #[cfg(any(feature = "ssr", feature = "csr"))]
 use crate::html::{AnyScope, Scope};
-use crate::html::{BaseComponent, NodeRef};
+#[cfg(feature = "csr")]
+use crate::html::{NodeRef, Scoped};
 #[cfg(feature = "ssr")]
 use crate::platform::io::BufWriter;
 
 /// A virtual component.
 pub struct VComp {
     pub(crate) type_id: TypeId,
     pub(crate) mountable: Box<dyn Mountable>,
-    pub(crate) node_ref: NodeRef,
```

Comment:
I don't know why is this the case, but the bundle size increase is likely due to the removal of this field.

Reply:
I think this has something to do with the size of the struct, you can try to replace it with `_marker: usize` and set it  the value to `0` in the constructor it should also work.

Reply:
I ran twiggy and it seems like most of bytes are coming from code that is untouched here:

```
yew/examples/counter_functional [begone-comp-node-ref]〉twiggy diff dist/counter_functional-931e7c89893382d7_bg.wasm dist-pr/counter_functional-ad2bf05e16727653_bg.wasm -n 10
 Delta Bytes │ Item
─────────────┼───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
        +510 ┊ yew::dom_bundle::blist::<impl yew::dom_bundle::traits::Reconcilable for yew::virtual_dom::vlist::VList>::reconcile::h897bdbe92f15f88d
        -285 ┊ yew::dom_bundle::bcomp::<impl yew::dom_bundle::traits::Reconcilable for yew::virtual_dom::vcomp::VComp>::attach::hbd2d9621b55add0b
        +253 ┊ <yew::html::component::lifecycle::CompStateInner<COMP> as yew::html::component::lifecycle::Stateful>::view::h68b91f18b2519e4c
        +234 ┊ yew::dom_bundle::bcomp::<impl yew::dom_bundle::traits::Reconcilable for yew::virtual_dom::vcomp::VComp>::attach::hde8285620aedef9a
        +150 ┊ yew::dom_bundle::bsuspense::<impl yew::dom_bundle::traits::Reconcilable for yew::virtual_dom::vsuspense::VSuspense>::attach::hb3a4b9fd00446556
        +144 ┊ yew::dom_bundle::bnode::<impl yew::dom_bundle::traits::Reconcilable for yew::virtual_dom::vnode::VNode>::reconcile_node::h2888f3017cf1b126
        +123 ┊ yew::dom_bundle::blist::BList::apply_unkeyed::hf07a69927390bfbe
         +87 ┊ <core::iter::adapters::zip::Zip<A,B> as core::iter::traits::iterator::Iterator>::next::hf228331121bfac52
         +60 ┊ yew::dom_bundle::bportal::<impl yew::dom_bundle::traits::Reconcilable for yew::virtual_dom::vportal::VPortal>::attach::h22b2b8a5f486b6e5
         +53 ┊ <yew::html::component::lifecycle::RenderRunner as yew::scheduler::Runnable>::run::h84768ff9205d0ffa
         +23 ┊ ... and 17 more.
       +1352 ┊ Σ [27 Total Rows]
```

not sure what's happening here

Reply:
Yes, it's weird. But increase the size of this struct does work.

```
376K with-marker/router/dist/router-872e26b381599680_bg.wasm
384K without-marker/router/dist/router-514df72ff86f280e_bg.wasm
```